Implement token --by-acct

vincentmele opened the pull request:
Fixes Issue #3964 where `chifra token --by-acct` was not implemented.

## Description
Implemented the --by-acct option for `chifra token` to the documentation explanation.

vincentmele committed:
Implement token --by-acct 

Fixes Issue #3964 where `chifra token --by-acct` was not implemented.

Signed-off-by: Vincent Mele <vincentmele@users.noreply.github.com>

tjayrush commented:
This is excellent. Thanks so much.

Ignore the failed test run below (it's failing for a different reason), but I ran the tests locally, and it is failing one of the tests.

```
chifra tokens  0xb9e7f8568e08d5659f5d29c4997173d84cdf2607 0xa25d04fa348f107428708ba19c95d1c79b3744bf 4285364 3510999 2510010 --by_acct
```

It produces this result:

```
blockNumber	holder	address	name	symbol	decimals	balance	balanceDec
4285364	0xa25d04fa348f107428708ba19c95d1c79b3744bf	0xb9e7f8568e08d5659f5d29c4997173d84cdf2607	Swarm City Token	SWT	18	435000000000000000000000	435000
3510999	0xa25d04fa348f107428708ba19c95d1c79b3744bf	0xa25d04fa348f107428708ba19c95d1c79b3744bf			18	0	0
2510010	0xa25d04fa348f107428708ba19c95d1c79b3744bf	0xa25d04fa348f107428708ba19c95d1c79b3744bf			18	0	0
```

Which is wrong, I believe (and please correct me if I'm wrong). It should be:

```
blockNumber	holder	address	name	symbol	decimals	balance	balanceDec
4285364	0xa25d04fa348f107428708ba19c95d1c79b3744bf	0xb9e7f8568e08d5659f5d29c4997173d84cdf2607	Swarm City Token	SWT	18	435000000000000000000000	435000
3510999	0xa25d04fa348f107428708ba19c95d1c79b3744bf	0xb9e7f8568e08d5659f5d29c4997173d84cdf2607	Swarm City Token	SWT	18	0	0
2510010	0xa25d04fa348f107428708ba19c95d1c79b3744bf	0xb9e7f8568e08d5659f5d29c4997173d84cdf2607	Swarm City Token	SWT	18	0	0
```

With multiple blocks specified on the command line, it appears to forget which one is the token.

tjayrush commented:
I'll merge it if you correct that issue. Thanks so much, again.

vincentmele committed:
Correct location of loop to enable token --by_acct and multiple blocks

Signed-off-by: Vincent Mele <vincentmele@users.noreply.github.com>

vincentmele commented:
Fixed. Added example below for byAcct with multiple tokens and multiple blocks.

```bash
chifra tokens  0xb9e7f8568e08d5659f5d29c4997173d84cdf2607 0x58b6A8A3302369DAEc383334672404Ee733aB239 0xa25d04fa348f107428708ba19c95d1c79b3744bf 4285364 3510999 2510010 latest --by_acct
```
should return
```bash
blockNumber     holder  address name    symbol  decimals        balance balanceDec
4285364 0xa25d04fa348f107428708ba19c95d1c79b3744bf      0xb9e7f8568e08d5659f5d29c4997173d84cdf2607      Swarm City Token        SWT     18      435000000000000000000000        435000
3510999 0xa25d04fa348f107428708ba19c95d1c79b3744bf      0xb9e7f8568e08d5659f5d29c4997173d84cdf2607      Swarm City Token        SWT     18      0       0
2510010 0xa25d04fa348f107428708ba19c95d1c79b3744bf      0xb9e7f8568e08d5659f5d29c4997173d84cdf2607      Swarm City Token        SWT     18      0       0
21872537        0xa25d04fa348f107428708ba19c95d1c79b3744bf      0xb9e7f8568e08d5659f5d29c4997173d84cdf2607      Swarm City Token        SWT     18      0       0
4285364 0xa25d04fa348f107428708ba19c95d1c79b3744bf      0x58b6a8a3302369daec383334672404ee733ab239      Livepeer Token  LPT     18      0       0
3510999 0xa25d04fa348f107428708ba19c95d1c79b3744bf      0x58b6a8a3302369daec383334672404ee733ab239      Livepeer Token  LPT     18      0       0
2510010 0xa25d04fa348f107428708ba19c95d1c79b3744bf      0x58b6a8a3302369daec383334672404ee733ab239      Livepeer Token  LPT     18      0       0
21872537        0xa25d04fa348f107428708ba19c95d1c79b3744bf      0x58b6a8a3302369daec383334672404ee733ab239      Livepeer Token  LPT     18      2236860321384596495     2.236860321384596495
```

tjayrush commented:
I fixed it. Thanks. I will close this PR and merge the corrected one #3966. This will be included in the next release to `master`, but you can use it from `develop` for now.
